rbd/group.go

```diff
@@ -160,7 +163,8 @@ func GroupImageRemoveByID(groupIoctx *rados.IOContext, groupName string,
 		cephIoctx(groupIoctx),
 		cGroupName,
 		cephIoctx(imageIoctx),
-		cid)
+		cid,
+		C.uint32_t(0))
```

Comment:
This change is part of the [in-progress ceph PR](https://github.com/ceph/ceph/pull/53793/files) that has mirror group APIs